modules/ROOT/pages/data-modeling.adoc

```diff
@@ -7,25 +7,22 @@ ifdef::prerelease[:page-status: {prerelease}]
 
 include::partial$_set_page_context.adoc[]
 
-
 // BEGIN -- Page Attributes
 :xref-sdk-nodejs-pg-start: xref:nodejs-sdk:hello-world:start-using-sdk.adoc[Node.js Server SDK]
 :xref-sdk-java-encryption: xref:java-sdk:concept-docs:encryption.adoc#format[field encryption format]
 // END -- Page Attributes
 
-
 // BEGIN -- Page Heading
 :param-abstract!:
 :param-related:  {revisions--xref} | {managing-tombstones--xref}
 include::partial$_show_page_header_block.adoc[]
 // END -- Page Heading
 
-
 == Introduction
+
 The guidance and constraints documented here relate to the design of data buckets and documents that you require, or may potentially require, to replicate using Sync Gateway functionality.
```

Comment:
Simplify

```suggestion
This page includes guidance and constraints relating to the design of data buckets and documents that you want to replicate using Sync Gateway.
```

Reply:
Done. I'm not sure I touched these pages though?